graphene_linked_events/schema.py

```diff
@@ -472,6 +479,32 @@ def resolve_keywords(parent, info, **kwargs):
         response = api_client.list("keyword", filter_list=kwargs)
         return json2obj(format_response(response))
 
+    @staticmethod
+    def resolve_popular_kultus_keywords(parent, info, **kwargs):
+        """Returns Kultus related keywords ordered by popularity.
+
+        Fetches keywords for keyword sets: category, additional criteria and
+        target group. Keywords are ordered in descending order by the number of events
+        and returned.
+        """
+        # TODO optimize and cache ?
+        set_ids = [
+            KEYWORD_SET_ID_MAPPING["CATEGORY"],
+            KEYWORD_SET_ID_MAPPING["ADDITIONAL_CRITERIA"],
+            KEYWORD_SET_ID_MAPPING["TARGET_GROUP"],
+        ]
+        keywords = []
+
+        for set_id in set_ids:
+            response = api_client.retrieve(
+                "keyword_set", set_id, params={"include": "keywords"}
+            )
+            response = json2obj(format_response(response))
+            keywords += response.keywords
+
+        keywords = sorted(keywords, key=lambda x: x.n_events, reverse=True)
```

Comment:
Duplicates should be filtered out. It's not an issue in a staging or production environment, but it is in my local :) 
Maybe with:
```
unique_keywords = list(set(keywords))
keywords = sorted(unique_keywords, key=lambda x: x.n_events, reverse=True)
```